packages/jellyfish-api-core/__tests__/category/blockchain/getBlockHash.test.ts

```diff
@@ -30,6 +30,7 @@ describe('BlockHash', () => {
     expect.assertions(1)
     try {
       await client.blockchain.getBlockHash(10000)
+      /* eslint-disable-next-line @typescript-eslint/no-explicit-any */
     } catch (err: any) {
```

Comment:
For `err` object, I feel like we should allow `any` to be used as the variable type can be unpredictable.
`Unknown` type could be better, but I don't want to refactor too much, so for now I keep it as `any`